convert2rhel/backup.py

```diff
@@ -102,10 +102,17 @@ def _install_local_rpms(self, pkgs_to_install, replace=False, critical=True):
         cmd = cmd_param + pkgs_to_install
         output, ret_code = run_subprocess(cmd, print_output=False)
         if ret_code != 0:
-            pkgs_as_str = " ".join(pkgs_to_install)
+            pkgs_as_str = utils.format_sequence_as_message(pkgs_to_install)
             loggerinst.debug(output.strip())
             if critical:
-                loggerinst.critical("Error: Couldn't install %s packages." % pkgs_as_str)
+                loggerinst.critical_no_exit("Error: Couldn't install %s packages." % pkgs_as_str)
+                raise exceptions.CriticalError(
+                    id_="FAILED_TO_INSTALL_PACKAGES",
+                    title="Couldn't install packages.",
+                    description="While attempting to roll back changes, we encountered an unexpected failure while attempting to reinstall one or more packages that we removed as part of the conversion.",
+                    diagnosis="Couldn't install %s packages. Command: %s Output: %s Status: %d"
+                    % (pkgs_as_str, cmd, output, ret_code),
```

Comment:
Do we need to output the command used here?

Reply:
True, I wonder if the command is useful here at all 🤔

Reply:
I've been assuming that 'diagnosis' is something intended for a developer, which may be a misunderstanding on my part. My main rationale for including the command was that this is a "can't happen" error--we couldn't think of a way to cause it to happen for a test--so I wanted to be clear on what it was that failed, exactly.

Reply:
But that's a good point, @jochapma... I don't think we ever discussed the intent of the messages we are producing. I, for instance, always assumed that they would be user oriented rather than developer oriented. Maybe we could bring this discussion to our Slack channel and check what others might think about that?